src/span.js

```diff
@@ -18,6 +100,24 @@ export default class Span {
     // ---------------------------------------------------------------------- //
 
     /**
+     * Returns the SpanContext object associated with this Span.
+     *
+     * @return {SpanContext}
+     */
+    context() {
+        if (API_CONFORMANCE_CHECKS) {
+            if (arguments.length !== 0) {
+                throw new Error('Invalid number of arguments');
+            }
+        }
+        if (this._imp) {
+            return new Tracer(this._imp.context());
+        }
+        // XXX: what to do here?
```

Comment:
(this is also known-wrong... I'm more concerned with the user-facing API for this stage of the PR)